source/elements/oneDAL/source/index.rst

```diff
@@ -4,13 +4,13 @@
 .. _oneDAL-section:
 
 ====================================
- |dal_full_name| (|dal_short_name|)
+ oneAPI |dal_full_name| (|dal_short_name|)
```

Comment:
I'd change the definition of |dal_full_name| instead, as "oneAPI" is a part of oneDAL's full name

Reply:
Yes, i also would like to do this. But there is common place for all components and it will be good to have consistency there.  However i don't think this is good idea to change this for all other components.  Think this will results in several iterations of discussions

Reply:
Full names of other components, I think, have the same "oneAPI" prefix, so it would make sense to update them all.

Reply:
yes, but not 1 day prior to CF =)

Reply:
ok, but I'll create an issue for Robert so he could set the timeline for this change

Reply:
https://github.com/oneapi-src/oneapi-spec/issues/87

Reply:
fine with me to go w/o oneAPI now and fix it later